tests/ParserSpec.hs

```diff
@@ -91,13 +94,114 @@ parseNames = do
     negative "four components" $ parseQualifiedName def "a.b.c.d"
 
 parseCompile :: ParseEnv -> Text -> Either String (Either PactError [Term Name])
-parseCompile pe m = compileExps pe mkEmptyInfo <$> (parseExprs m)
+parseCompile pe m = compileExps pe mkEmptyInfo <$> parseExprs m
+
+rightRight :: Either l1 (Either l2 r) -> Bool
+rightRight (Right Right {}) = True
+rightRight _ = False
```

Comment:
I like `is (_Right . _Right)` for this, idk what the style is here.

Reply:
Since we're using it as a functional completion in `shouldSatisfy`, it's a little more ergonomic here to have the function definition imo. Tho, absolutely, testing nested cases with `is` is quite nice.